src/DynamoCore/Logging/DynamoLogger.cs

```diff
@@ -139,6 +141,8 @@ public string LogText
             }
         }
 
+        internal HashSet<NotificationMessage> Notifications { get; }
```

Comment:
this definitely seems weird - so it's a cache of notifications and only useful if the notifications extension is not loaded?

Reply:
Yes.